dsms/core/configuration.py

```diff
@@ -74,10 +74,31 @@ class Configuration(BaseSettings):
         description="JWT bearer token for connecting to the DSMS instance",
     )
 
+    enable_auto_reauth: bool = Field(
+        True,
+        description="""Whether to reauthenticate with username and password
```

Comment:
"Whether to **automatically** reauthenticate"

Reply:
Done.